tests/MongoDB.Driver.Examples/ChangeStreamExamples.cs

```diff
@@ -44,10 +45,10 @@ public void ChangeStreamExample1()
             .Start();
 
             // Start Changestream Example 1
-            var enumerator = inventory.Watch().ToEnumerable().GetEnumerator();
-            enumerator.MoveNext();
-            var next = enumerator.Current;
-            enumerator.Dispose();
+            var cursor = inventory.Watch();
+            while (cursor.MoveNext() && cursor.Current.Count() == 0) { }
```

Comment:
Hm, do you think the comment from example 3 might be helpful here too?  😃 
`// keep calling MoveNext until we've read the first batch`

Reply:
Done